src/chi/components/dropdown/dropdown.scss

```diff
@@ -0,0 +1,130 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+  .m-dropdown {
+    position: relative;
+    display: inline-block;
+
+    .m-dropdown__menu {
+      position:absolute;
+      left: 0;
+      min-width: 100%;
+    }
+  }
+
+  .m-dropdown__trigger {
+    background-image: url('./icon-chevron-thin-down.svg');
```

Comment:
Any thoughts on rendering the arrow as an ::after pseudo element so we can apply animations to it?